frontend/src/scenes/sessions/SessionsPlayerDrawer.tsx

```diff
@@ -21,11 +21,11 @@ export default function SessionsPlayerDrawer(): JSX.Element {
             footer={
                 <>
                     {nav.prev && (
-                        <Button style={{ marginRight: 12 }} onClick={() => loadSessionPlayer(nav.prev)}>
+                        <Button style={{ marginRight: 12 }} onClick={() => nav.prev && loadSessionPlayer(nav.prev)}>
```

Comment:
nav.prev is already being checked above in `nav.prev && (` - shouldn't that constrain things?

Reply:
Yeah, but TSC didn't pick that up :(.

It's smart with if statements, but not with JSX it seems.